src/views/Prompt.js

```diff
@@ -0,0 +1,97 @@
+const MissionUtils = require('@woowacourse/mission-utils');
```

Comment:
구조분해 할당으로 선언했다면 좋았을 것 같습니다.
```suggestion
const { Console } = require('@woowacourse/mission-utils');
```

Reply:
구조분해로 수정해야겠습니다 👍👍